state_experiments/plot_state_results.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pickle
+import pandas as pd
+import seaborn as sns
+from gerrychain import Graph, GeographicPartition, Partition, Election, accept
+from gerrychain.updaters import Tally, cut_edges
+import glob
+import functools
+import operator
+import argparse
+
+parser = argparse.ArgumentParser(description="Plot State Run Results", 
+                                 prog="plot_state_results.py")
+parser.add_argument("state", metavar="state id", type=str,
+                    choices=["VA", "TX", "CO", "LA", "NM"],
+                    help="which state to run chains on")
+parser.add_argument("race_col", metavar="Race_column", type=str,
+                    choices=["BVAP", "HVAP", "nWVAP"],
+                    help="Which race_col")
+args = parser.parse_args()
+
+
+num_h_districts = {"VA": 100, "TX": 150, "AR": 100, "CO": 65, "LA": 105, "NM": 70}
+pop_bal = {"CO": 2.0, "LA": 4.5, "NM": 4.5, "TX": 2.0, "VA": 2.0}
+
+
+ST = args.state
+NUM_DISTRICTS = num_h_districts[ST]
+MIN_COL = args.race_col
+iters = 100000
+# POP_COL = "TOTPOP"
+# N_SAMPS = 10
+
+# EPS = 0.045
+
+
+def foldl(func, acc, xs):
+  return functools.reduce(func, xs, acc)
+
+foldr = lambda func, acc, xs: functools.reduce(lambda x, y: func(y, x), xs[::-1], acc)
+
+def get_state_runs(state, seats, iters=100000, pop_bal=2.0, min_col="nWVAP",
+                   ls=[2,5,10,20,40,80], ps=[0.25, 0.125, 0.0625]):
```

Comment:
Could you leave a docstring for the functions explaining what the arguments are?